Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "core-text"
-version = "8.0.0"
+version = "8.0.1"
```

Comment:
What about the version number this time? I can't find any re-exports of `core-foundation`. On the other hand the `impl TCFType for CTFont` and similar do change, so I guess it's an API change? Even if very few are likely to rely on exactly those.

Reply:
Since we use the types from core-foundation and core-graphics in the public API of this crate, I believe that means we need to increase this to 9.0.0.